src/fileinfo/file_detector/macho_detector.cpp

```diff
@@ -118,6 +119,10 @@ void MachODetector::getSections()
 		fsec.setMemoryAlignment(pow(2, mSec->getAlignment()));
 		fsec.setRelocationsOffset(mSec->getRelocationOffset());
 		fsec.setNumberOfRelocations(mSec->getNumberOfRelocations());
+		if(sec->getEntropy(entropy))
```

Comment:
I suggest defining `double entropy;` right before `if(sec->getEntropy(entropy))`, not 14 lines before. It is generally a good practice to define variables as close to their first use, especially when they are not initialized.